databases/week3/querries.sql

```diff
@@ -0,0 +1,95 @@
+--Meals Querries
+--Get all meals
+Select*
+from meal;
+--Add a new meal 
+insert into meal
+values(6, 'A Vegetarian Meal', '2 Salad, 2 Soup, 2 Starter, 2 Main Dish, 1 Desert', 'south-west', '2020-10-16', 10, 500, '2020-10-20');
```

Comment:
this sql will not work, because when you use insert you have also to provide columns names that values will be store. Fx:
INSERT INTO meal(
id,
title,
description,
location,
when,
max_reservations,
price,
created_date
)
VALUES (
'3',
'Gelée de caviar à la crème de chou-fleur',
'Caviar jelly with cauliflower cream',
'Enghavevej 80, 2450 København',
'2019-10-15 09:00:09',
'300',
'80',
'2019-10-15 07:05:09'
);